### DISCUSSION TITLE
How to avoid any?

### DISCUSSION THREAD

**Discussion by ccfiel** [hayes/pothos#1010] [Q&A] [answered]
I do not want to disable  @typescript-eslint/no-explicit-any. Any ideas?

![SCR-20230904-lvjf](https://github.com/hayes/pothos/assets/1788360/c18ff00e-6b0c-449b-ba69-27905dcbf426)

```
const yoga = createYoga({
  graphqlEndpoint: '/',
  landingPage: false,
  schema,
  context: async ({ req }: any) => ({
    currentUser: await getUserFromAuthHeader(req.headers.authorization),
  }),
});
```

**Comment by hayes** [comment, marked as answer]
This is probably more of a question for yoga, and I think it depends on the runtime you are using, but I think you can use `ServerRequest` if you install @types/node
